RealmTasks Apple/RealmTasks iOS/ViewController.swift

```diff
@@ -267,15 +283,23 @@ final class ViewController<Item: Object, Parent: Object where Item: CellPresenta
             if bottomViewController === parentViewController?.childViewControllers.last {
                 finishMovingToNextViewController(.Down)
             } else {
-                try! items.realm?.write {
-                    let itemsToDelete = items.filter("completed = true")
-                    let numberOfItemsToDelete = itemsToDelete.count
-                    guard numberOfItemsToDelete != 0 else { return }
+                items.realm?.beginWrite()
```

Comment:
it makes the `cancelWrite()` scenario less nice, but I can change the commit to use `finishUIWrite()`